internals/daemon/access.go

```diff
@@ -68,3 +68,18 @@ func (ac UserAccess) CheckAccess(d *Daemon, r *http.Request, user *UserState) Re
 	// An identity explicitly set to "access: untrusted" isn't allowed.
 	return Unauthorized(accessDenied)
 }
+
+// MetricsAccess allows requests over the HTTP from authenticated users.
```

Comment:
Although an acronym (well, technically an initialism), I think 'http' is in such wide use that it's become its own proper noun, so the 'the' can probably be dropped